packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -13,6 +13,8 @@ import 'src/common.dart';
 export 'src/common.dart';
 export 'widgets.dart';
 
+enum SignInOption { defaultSignIn, gamesSignIn }
```

Comment:
Maybe call these `standard` and `games`. The `SignIn` part is already given by the enum type name.